.travis.yml

```diff
@@ -19,7 +19,6 @@ rvm:
   - 2.2.3
   - 2.2.4
   - 2.3.0
-  - jruby-9000
```

Comment:
I removed because jruby-9000 is broken due to BigDecimal, but I have to add the right version